chpl/chpl-api/src/main/java/gov/healthit/chpl/web/controller/ActivityController.java

```diff
@@ -809,6 +810,18 @@ public ActivityDetails activityById(@PathVariable("id") final Long id)
         return activityManager.getApiKeyActivity(startDate, endDate);
     }
 
+    @ApiOperation(value = "Get auditable data for all API keys",
+        notes = "Users must specify 'start' and 'end' parameters to restrict the date range of the results."
+            + "Security Restrictions: Only ROLE_ADMIN or ROLE_ONC")
+    @RequestMapping(value = "/metadata/api-keys", method = RequestMethod.GET, produces = "application/json; charset=utf-8")
+    public List<ActivityDetails> metadataForApiKeys(@RequestParam final Long start,
```

Comment:
Based on our phone conversation...  This needs to be updated to return an "ApiActivityMetadata" object.